### ISSUE TITLE
Server Closes when input is given

### ISSUE THREAD

**Issue by srikrishna123g** [dineshjoshy/Chatbot_Project#2]
When an input is given on the website the server Closes with this output in the terminal.

('::1', 65370, 0, 0) connected
('::1',65370, 0, 0) closed

**Comment by loubna1998** [comment]
same here :(

**Comment by Manav1918** [comment]
Same issue...
Server is not giving any response 
When I send any data, server closes

**Comment by loubna1998** [comment]
hello @Manav1918  i solved the problem, so the thing is that you need to make sure that the "get-response()" function in the server.py file is the one in the chatbot.py file. in my case i just moved to code in server.py to chatbot.py and made 1 file instead of 2. don't forget to remove the "from chatbot import get_response" line while doing that.
[chatbot.txt](https://github.com/dineshsambasivam/Chatbot_Project/files/3284864/chatbot.txt)

**Comment by Manav1918** [comment]
It's not working yet. May i have to run chatbot.train.py first?

Pawan Kumar Prachi

**Comment by Manav1918** [comment]
I have downloaded and pasted the code u sent and renamed it to chatbot.py
from chatbot.txt . And removed older chatbot. Py file and server. Py file
too.
But problem is still  existing.
Help me plz
Pawan Kumar Prachi

**Comment by loubna1998** [comment]
yes @Manav1918  you have to run chatbot.train.py first then run chatbot.py and open the html file on a browser

**Comment by Manav1918** [comment]
I was doing as u said.... But there is still d problem. When I am typing
hello in browser, connection closed method runs in console and then nothing
happens

Pawan Kumar Prachi

**Comment by YugamBajaj** [comment]
can someone please share chatbot_train.py file !

**Comment by YugamBajaj** [comment]
![image](https://user-images.githubusercontent.com/48938476/59668492-3b928e00-91d6-11e9-8265-826846b9684d.png)

i am getting the same error @Manav1918 
@dineshsambasivam @loubna1998 pls help

**Comment by Manav1918** [comment]
I left this and used official example to learn. That's working fine

Pawan Kumar Prachi

**Comment by YugamBajaj** [comment]
Can you please re-share the files which are working for you. @Manav1918

**Comment by Manav1918** [comment]
Go to chatterbox official link in GitHub and download git of example for
chatbot with flask

Pawan Kumar Prachi

**Comment by YugamBajaj** [comment]
Please share the link here. It will be helpful for all.

**Comment by YugamBajaj** [comment]
Please share the link here, it will be helpful for all.
@Manav1918

**Comment by Manav1918** [comment]
https://github.com/chamkank/flask-chatterbot

Pawan Kumar Prachi

**Comment by Manav1918** [comment]
If it will not work for anyone, stay tune to youtube.com/c/Cidaneducationhub,
I will make a tutorial on flask  chatterbot ASAP. Bacause I am seeing, many
people have problems in making such app...

Pawan Kumar Prachi

**Comment by Deepika2010** [comment]
> hello @Manav1918 i solved the problem, so the thing is that you need to make sure that the "get-response()" function in the server.py file is the one in the chatbot.py file. in my case i just moved to code in server.py to chatbot.py and made 1 file instead of 2. don't forget to remove the "from chatbot import get_response" line while doing that.
> [chatbot.txt](https://github.com/dineshsambasivam/Chatbot_Project/files/3284864/chatbot.txt)



> same here :(

Same here ...pls help me to solve
